packages/ui/components/dialog/Dialog.tsx

```diff
@@ -175,6 +232,22 @@ DialogContent.displayName = "DialogContent";
 export const DialogTrigger = DialogPrimitive.Trigger;
 // export const DialogClose = DialogPrimitive.Close;
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function DialogCloseWrapper({ children, ...props }: any) {
```

Comment:
we don't prefer to use 'any' type in the codebase

Reply:
can you replace it with actual type?

Reply:
I have replaced it. Thanks for reviewing.